example/index.js

```diff
@@ -1,5 +1,3 @@
-/* global Worker:false, fetch:false, window:false, document:false */
```

Comment:
yes, this is why this second commit is prefixed with `fix:` since it's fixing the regression :)
i can't imagine why someone would revert to the previous commit

Reply:
squashed

Reply:
>  i can't imagine why someone would revert to the previous commit

Indeed, this won't most probably ever happen. Except maybe when using `git bisect` in which case having a monotonic (ie. without regression) history really helps.

Squashing this also might be good for any future developer looking at the `fix` commit wondering why it wasn't fixed earlier.

> squashed

Thanks a lot!